packages/trimerge-sync/src/AbstractLocalStore.ts

```diff
@@ -34,6 +34,16 @@ export type BroadcastEvent<CommitMetadata, Delta, Presence> = {
   remoteOrigin: boolean;
 };
 
+export type EventChannel<CommitMetadata, Delta, Presence> = {
```

Comment:
should this be imported into trimerge-indexed-db?

Reply:
I think we need it to unless we want to create a separate package for broadcastchanneleventchannel or allow dom types in trimerge-sync. This feels like the right thing to do.